package-overrides/github/allpro/layout@1.4.3.json

```diff
@@ -5,11 +5,12 @@
         "lib": "source/stable"
     },
     "dependencies": {
-        "jquery": "*"
+        "github:components/jquery": "*",
+        "github:components/jqueryui": "*"
```

Comment:
just out of curiousity is this something new ? because most of the other overrides use something like :
```
"dependencies": {
        "angular": "^1.3.2",
        "angular-ui-router": "0.2.10"
    }
```